_posts/2016-03-15-a-new-kind-of-open-source-event.md

```diff
@@ -0,0 +1,72 @@
+
+Additionally, we ask you to add this banner on top of the starter issues, so that others know that the issue is reserved for participants of the “Welcome to Open Source, Atlanta” event:
+
+<img src="/blog/images/201603/welcome-to-open-source-banner.png" alt="Community">
+
+Simply copy and paste this code and put it on top of every issue you create for the event:
+
+    ![welcome-to-open-source-atlanta](https://raw.githubusercontent.com/welcome-to-open-source/Atlanta/master/assets/banner.png)
+
+    **This issue is reserved for participants of [Welcome to Open Source, Atlanta](http://www.meetup.com/Jr-Dev-Mentoring/events/228951389/)**.
+
+    If it’s still available after April 2nd, it’s all yours :)
+
+    ---
+
+## Be available for questions and reviews of pull requests
+
+**We will be there to help** the participants with all questions regarding _git_ and GitHub, but if there are follow up questions to your prepared issues, it would be great if someone from the project could be available online or in person to answer them. And if a pull request is ready, it would be fabulous if the pull request could be reviewed in a timely manner.
+
+# The event
+
+Before the event, we will send out an email with links to introductions to _git_, GitHub, and Open Source and we will ask the participants to install git on their laptops ahead of time.
+
+On the day of the event, each participant will add themselves to [our event page on GitHub](https://github.com/welcome-to-open-source/Atlanta) using a pull request. **This is to get their feet wet in a secure environment**.
+
+Once that is accomplished, participants will look through the list of prepared issues by the participating projects and comment on the ones they want to pursue.
+
+The hosts will be there to help with any questions regarding _git_ and GitHub, and encourage the participants to work together and help each other out.
+
+# After the event
+
+All prepared issues that have not been worked on during the event will be free for everyone to take. We hope that the participating projects will find joy in preparing starter issues, even without an event happening, to be able to welcome new contributors to Open Source at all times.
+
+If you would like to host a “Welcome to Open Source” event in your city, [let us know](http://hood.ie/contact/)! We are more than happy to assist.
+
+Thanks to Alex, Jan & Jenn for their reviews!
+
+The [online Word editor](http://wordhtml.com/ "Online Word Editor") will help you compose easily documents for the web.
```

Comment:
LOL I used this editor in the process from Google Doc -> Markdow. What a scam to their ad in there, I totally missed it. Thanks for catching it :)